CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#619](https://github.com/rubocop/rubocop-rails/pull/619): Fix method shadowing check for `Rails/ReadWriteAttribute` cop. ([@nvasilevski][])
```

Comment:
Can you write to the following instead of editing the changelog directly?

> * [x] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop-rails/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

Reply:
Done, sorry, should have done it from the beginning 
CI seems to be read because of some github actions being down?

Reply:
It may be a GitHub Actions incident. I'll take a look and merge if it works. Thank you!

Reply:
It may be this one.
https://www.githubstatus.com/incidents/cypv026dr23w